smartcontractkit/external-adapters-js
# Refactor Anchor EA to just use single BigNumber library

## Closes 31409

## Description

This PR adds several improvements to the Anchor EA to pull `bETH` and `bLUNA` prices

 - Fetch `ETH-stETH` exchange rate by calling `get_virtual_price` on the Curve pool contract
 - Refactor EA to only use the `BigNumber` library exposed by `ethers`
 - Pull intermediary token prices from on chain feeds instead of directly from data providers
 - Update EA to return string representation of result

## Changes

- Update Anchor EA to pull intermediary token prices from on chain feeds
- Get `stETH/ETH` exchange rate by calling `get_virtual_price` function on Curve pool contract
- Update Anchor EA to use fixed point decimal math to handle decimal precision
- Update EA to return a string representation of the BigNumber result



## Steps to Test (WIP)

1. Run EA and verify that it is working
2. Ensure unit tests pass
3. Ensure integration tests pass

## Quality Assurance
